pinot-common/src/main/java/com/linkedin/pinot/common/utils/FileUploadUtils.java

```diff
@@ -43,6 +47,14 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(FileUploadUtils.class);
   private static final String SEGMENTS_PATH = "segments";
+  public static final String UPLOAD_TYPE = "UPLOAD_TYPE";
+  public static final String DOWNLOAD_URI = "DOWNLOAD_URI";
+  private static final HttpClient FILE_UPLOAD_HTTP_CLIENT = new HttpClient();
```

Comment:
If the same instance of HttpClient is reused, then a multithread-aware connection manager should be used. See http://hc.apache.org/httpclient-3.x/threading.html